frontend/src/app/Settings/Users/UserDetail.tsx

```diff
@@ -148,43 +358,13 @@ const UserDetail: React.FC<Props> = ({
       <div>
         <h2 className="display-inline-block margin-top-1 margin-bottom-0 user-name-header">
           {displayFullName(user.firstName, user.middleName, user.lastName)}
-          {user?.id === loggedInUser.id ? (
-            <span className="usa-tag margin-left-1 bg-base-lighter text-ink">
-              YOU
-            </span>
-          ) : null}
+          {displayYou()}
         </h2>
-        <div className="user-status-subheader">{statusText}</div>
+        <div className="user-status-subheader">{getUserStatusText()}</div>
       </div>
       <div className="user-header grid-row flex-row flex-align-center">
-        {user.status === "SUSPENDED" ? (
-          <>
-            <div className="status-tagline">
-              Users are deactivated after 60 days of inactivity.
-            </div>
-            <Button
-              variant="outline"
-              className="margin-left-auto margin-bottom-1"
-              onClick={() => updateShowReactivateUserModal(true)}
-              label="Activate user"
-              disabled={isUpdating}
-            />
-          </>
-        ) : null}
-        {user.status === "PROVISIONED" ? (
-          <>
-            <div className="status-tagline">
-              This user hasn’t set up their acccount.
-            </div>
-            <Button
-              variant="outline"
-              className="margin-left-auto margin-bottom-1"
-              onClick={() => updateShowResendUserActivationEmailModal(true)}
-              label="Send account setup email"
-              disabled={isUpdating}
-            />
-          </>
-        ) : null}
+        {displayUserSuspended()}
+        {displayUserProvisioned()}
```

Comment:
These two I would merge into a subcomponent <UserSpecialStatusNotice /> and because the DOM looks the same I suggest just changing the content of the status-tagline, the label of the button and the onclick.

`{user.status !== 'ACTIVE' && <SpecialStatusNotice />}`